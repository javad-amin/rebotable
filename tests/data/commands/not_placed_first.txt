MOVE
REPORT
PLACE 3,2,SOUTH
MOVE
REPORT
