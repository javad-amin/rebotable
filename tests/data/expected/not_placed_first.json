{
    "reports": [
        "3,1,SOUTH"
    ]
}
